Makefile

```diff
@@ -1,2 +1,12 @@
 build:
 	docker build -t socketplane/socketplane .
+
+test:
```

Comment:
tried to test this patch, though the basic functionality worked as expected, the test coverage fails due to :

sudo make test
go test -covermode=count -test.short -coverprofile=daemon.cover.out -coverpkg=./... ./daemon
daemon/bonjour.go:6:2: cannot find package "github.com/socketplane/socketplane/Godeps/_workspace/src/github.com/Sirupsen/logrus" in any of:
	/usr/lib/go/src/pkg/github.com/socketplane/socketplane/Godeps/_workspace/src/github.com/Sirupsen/logrus (from $GOROOT)
	($GOPATH not set)
daemon/daemon.go:12:2: cannot find package "github.com/socketplane/socketplane/Godeps/_workspace/src/github.com/codegangsta/cli" in any of:
	/usr/lib/go/src/pkg/github.com/socketplane/socketplane/Godeps/_workspace/src/github.com/codegangsta/cli (from $GOROOT)
	($GOPATH not set)
daemon/utils.go:9:2: cannot find package "github.com/socketplane/socketplane/Godeps/_workspace/src/github.com/docker/libcontainer/netlink" in any of:
	/usr/lib/go/src/pkg/github.com/socketplane/socketplane/Godeps/_workspace/src/github.com/dock
▽
er/libcontainer/netlink (from $GOROOT)
	($GOPATH not set)
daemon/api.go:9:2: cannot find package "github.com/socketplane/socketplane/Godeps/_workspace/src/github.com/gorilla/mux" in any of:
	/usr/lib/go/src/pkg/github.com/socketplane/socketplane/Godeps/_workspace/src/github.com/gorilla/mux (from $GOROOT)
	($GOPATH not set)
daemon/bonjour.go:7:2: cannot find package "github.com/socketplane/socketplane/Godeps/_workspace/src/github.com/socketplane/bonjour" in any of:
	/usr/lib/go/src/pkg/github.com/socketplane/socketplane/Godeps/_workspace/src/github.com/socketplane/bonjour (from $GOROOT)
	($GOPATH not set)
daemon/network.go:10:2: cannot find package "github.com/socketplane/socketplane/Godeps/_workspace/src/github.com/socketplane/ecc" in any of:
	/usr/lib/go/src/pkg/github.com/socketplane/socketplane/Godeps/_workspace/src/github.com/socketplane/ecc (from $GOROOT)
	($GOPATH not set)
daemon/bridge.go:16:2: cannot find package "github.com/socketplane/socketplane/Godeps/_workspace/src/github.com/socketplane/libovsdb" in any of:
	/usr/lib/go/src/pkg/github.com/socketplane/socketplane/Godeps/_workspace/src/github.com/socketplane/libovsdb (from $GOROOT)
	($GOPATH not set)
daemon/bonjour.go:8:2: cannot find package "github.com/socketplane/socketplane/datastore" in any of:
	/usr/lib/go/src/pkg/github.com/socketplane/socketplane/datastore (from $GOROOT)
	($GOPATH not set)
daemon/bridge.go:17:2: cannot find package "github.com/socketplane/socketplane/ipam" in any of:
	/usr/lib/go/src/pkg/github.com/socketplane/socketplane/ipam (from $GOROOT)
	($GOPATH not set)
make: *** [test] Error 1